bdi-frontend/src/appRouter.js

```diff
@@ -4,7 +4,7 @@ import { createRouter, createWebHistory } from "vue-router";
 import store from "./store/index";
 
 const routes = [
-  { path: "/login", component: LoginView },
+  { path: "/inloggen", component: LoginView },
```

Comment:
```suggestion
  { path: "/login", component: LoginView },
```
We houden dit Engels, geld ook voor de rest